client/src/components/Container.jsx

```diff
@@ -35,16 +41,59 @@ class Container extends React.Component {
         });
       });
   }
+
+  getReviewImages(storeId) {
+    axios.get(`/stores/${storeId}/review_images`)
+      .then((res) => {
+        this.setState({
+          reviewImages: res.data
+        });
+      });
+  }
+
+  getAverageStars(storeId) {
+    console.log(this.state.reviews);
+    var average;
+    var total = 0;
+    for (var i = 0; i < this.state.reviews.length; i++) {
+      total += reviews[i].stars;
+    }
+    average = Math.floor(total / this.state.reviews.length);
+    console.log(`average stars for store: ${average}`);
+  }
+
+  showMore() {
+    this.setState({
+      showMore: !this.state.showMore
+    });
+  }
+
+
   
   render() {
-    return (
-      <div style={styles.container}>
-        <h2 style={styles.header}>Reviews</h2>
-        <ReviewContainer showPrice="false"/>
-        <ReviewPhotosContainer openModal={this.toggleModal}/>
-        <Modal showModal={this.state.showModal} onClose={this.toggleModal}/>
-      </div>
-    );
+    if (!this.state.showMore) {
```

Comment:
Instead of having two return statements you can use the ternary operator if you want

Reply:
I might explore that if I have time!